spec.md

```diff
@@ -20,32 +20,28 @@ This Artifacts Spec provides a reference for artifact authors and registry imple
 
 ## Scope
 
-Create a first version of the Artifacts specification that is dependent on [OCI manifest][image-manifest] 1.0 to represent individual artifact types including [OCI Image][image-spec], [Helm][helm] and [Singularity][singularity].
-
-**v1.0 of the image-spec** defines index and manifest.
+Create a first version of Artifacts support which is dependent on [OCI manifest][image-manifest] 1.0 to represent individual artifact types including [OCI Image][image-spec], [Helm][helm] and [Singularity][singularity].
 
 <img src="./media/image-spec-objects.png" width=500x>
 
-**Artifacts 1.0** utilize manifest and index to represent multiple artifact types, including the [OCI Image-spec][image-spec]
-
 <img src="./media/artifacts-spec-objects.png" width=500x>
 
 ### Future Scope
```

Comment:
suggest ## Notice about Upcoming Changes

Reply:
To simplify things, I've removed these headings.